application/backend/.gitignore

```
@@ -14,5 +14,6 @@ wheels/
 TODO*
 
 data/
+docs/openapi.json
```

```suggestion
**/openapi.json
```
We should perhaps change the default target folder for `openapi.json`, because `backend/docs/` doesn't exist in the first place.